### PR TITLE
Added storybook and dependencies

### PR DESCRIPTION
**Pull request by mlipardo**

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  .add('Default', () => <DateTimePicker />);
+
+storiesOf('DropDownField', module)
+  .add('Default', () => (
+    <DropDownField
+      name="Characters"
+      options={['Mickey Mouse', 'Minnie Mouse', 'Goofy']}
+      selectedOption=""
+      handleOnChange={() => undefined}
+      placeholder="Choose Characeter"
```
Comment (edgibbs) [inline stories/index.js]: Typo on character.

**Comment by Sushmareddy-OSI** [comment]
+1 , Looks good.

**Comment by eckdanny-osi** [comment]
Depending on how to group responds to #8, we may want to _avoid_ tracking the [exported static files](https://storybook.js.org/basics/exporting-storybook/) in version control

**Comment by edgibbs** [comment]
So, an initial issue is Storybook generates a static site that then is going to be hosted as github pages on the project.  So I believe we have to have some of those static assets and it will probably get regenerated when we change components.  @NagaPradeepGajula-osi and I had a short conversation with the thought that we may be able to hide the changes to dist in the reviews.

**Comment by eckdanny-osi** [comment]
96e6c9b4ee822d14f65c06e49d787a6e81296f24 removes `.out` (default dir to which storybook generates static app) from version control) #8

**Pull request merged**